sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -506,14 +507,20 @@ def _create_table_properties(
 
             this = partitioned_by[0]
 
-            if isinstance(this, exp.Column):
-                if partition_interval_unit == IntervalUnit.MINUTE:
-                    raise SQLMeshError("BigQuery does not support partitioning by minute")
+            if isinstance(this, exp.Column) and partition_interval_unit != IntervalUnit.MINUTE:
```

Comment:
For the case when partition_by is a column and we get an interval unit of a minute, is the thought to use the BQ error message instead of our own?

Reply:
Yes, but the error message will only happen if the column is not date and the custom expression was not provided.

Reply:
If the data type is timestamp or datetime, only providing the column will fail. Does it make sense to provide a better error message to the user of what to do? If they are relying on our "magic", it would be helpful to point them at documentation that informs them what to do.

Reply:
I'd rather not hide BQ responses for now. If we can derive the partition automatically - great, if not - we should just let user interpret the error and provide a correct `partition_by` based on the error message they got from BQ.